### PR TITLE
Limit pagination include to index. Fix pagination calc

### PR DESCRIPTION
**Pull request by pharman**
Made a mistake about the pagination sorry. Must have been having a blonde moment

### CODE REVIEW

File: Resources/views/base_layout.html.twig [hunk 1]
```diff
@@ -12,7 +12,6 @@
                 <h1>{{ 'redirects' | trans }}</h1>
             </div>
             {% block armb_content %}{% endblock %}
-            {% block armb_pagination %}{% endblock %}
```
Comment (pharman) [inline Resources/views/base_layout.html.twig]: This should only be on the index page

**Comment by pharman** [comment]
I was trying to allow the pagination block to be overridden but it's a bit tricky as the inheritance is the wrong way around.

**Comment by llewellynthomas** [comment]
@pkraeutli could you please have a look at this pull request. Thanks

File: Resources/views/Mapping/index.html.twig [hunk 2]
```diff
@@ -81,6 +81,11 @@
                 {% endfor %}
             </tbody>
         </table>
+
+        {% block armb_pagination %}
```
Comment (pkraeutli) [inline Resources/views/Mapping/index.html.twig]: Why is this in a block?

Comment (pharman) [inline Resources/views/Mapping/index.html.twig]: Just to allow it to be overridden, not particularly useful any more though

**Pull request merged**